client/src/components/FrequentItemList.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import FrequentImages from './FrequentImages.jsx';
+import CheckboxContainer from './CheckboxContainer.jsx';
+
+const FrequentItemList = (props) => {
```

Comment:
Nice modularization of the subcomponents `FrequentImages` and `CheckboxContainer`